portal-web/test/functional/com/liferay/portalweb/macros/json/user/JSONUserGetter.macro

```diff
@@ -57,11 +50,11 @@ definition {
 	macro getEmailAddressNoSelenium {
 		Variables.assertDefined(parameterList = ${userId});
 
-		var curl = JSONUserGetter._getUserByIdCurlNoSelenium(
-			authenticationMethod = ${authenticationMethod},
-			userId = ${userId});
+		var curl = JSONUserGetter._getUserByIdCurlNoSelenium(userId = ${userId});
 
-		var userEmailAddress = JSONCurlUtil.get(${curl}, "$['emailAddress']");
+		var response = JSONCurlUtil.get(${curl});
```

Comment:
Seeing here using two lines to get userEmailAddress, wondering are we going to reduce using JOSNCurlUtil.get(${curl},"$..emailAddress")?

Reply:
this way it works, seems like the $['emailAddress']" part was a problem, I will refactor thanks!